### PR TITLE
chore: update dependencies

### LINKED ISSUE #739: Update examples to point at 'master'
After renaming branches examples point to the deprecated beta version on NPM. Update them to reference master.

### PR DESCRIPTION
**Pull request by marcelltoth**
Upgrades to Angular 11 and Stoplight Elements 6 stable

Resolves stoplightio/elements#739
Closes #34
Closes #37
Closes #38 

I had to also replace tslint by eslint. Our TSLint config has a peer-dependency on TS 4.0+ which is not trivially compatible with the Angular (10) tooling yet, blocking the update. As TSLint is deprecated anyway, I think this is an improvement. 

I could not use `@stoplight/eslint-config` directly however, because that has dependencies on `eslint-react` and friends, but I applied the prettier rules which can be used directly. I think that's more than enough for now.

### CODE REVIEW

File: .prettierrc.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    ...require("@stoplight/eslint-config/prettier.config"),
```
Comment (mpodlasin) [inline .prettierrc.js:2]: just curious - why is it reexported this way?

**Review by mpodlasin — APPROVED** [review]
I am not an angular expert, and even less an angular configuration expert, but nothing here raises any red flags for me.

If it works, it works ;)

**Review by mallachari — APPROVED** [review]
WFM!

File: .prettierrc.js (hunk 1 above)
Comment (marcelltoth) [reply .prettierrc.js:2]: If you mean why not
```js
module.exports = require("@stoplight/eslint-config/prettier.config")
```
then I don't know. This is how it's everywhere, probably the other one would work too but didn't try. I guess this gives us the opportunity to override some rules later on if needed.

This is prettier's config file, so essentially what we're doing here is telling it to inherit all configuration from our common configuration repo.

**Pull request merged**